cli/template/extras/src/trpc/server.ts

```diff
@@ -1,29 +1,60 @@
 import {
```

Comment:
should we `import "server-only"` here? I think it's the technically right thing to do so you can't import and attempt to use this in CCs

Reply:
yes seems reasonable